pages/common/nvim.md

```diff
@@ -1,7 +1,7 @@
 # nvim
 
 > Neovim, a programmer's text editor based on Vim, provides several modes for different kinds of text manipulation.
-> Pressing `i` in normal mode enters insert mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion.
+> Pressing `i` in normal mode enters insert mode. `<Esc>` goes back to normal mode, which doesn't allow regular text insertion. See also `vim`, `vimtutor`,`vimdiff`.
```

Comment:
can we not use another >

Reply:
Yeah, I too thought of adding it in the next line, due to the 3 line restriction I concatenated it. Maybe it's time to make it 4 lines so that some pages can have additional information.

Reply:
Done, Moved it to the following line @CleanMachine1.

Reply:
> can we not use another >

Only if style guide permits it. :) But in the current state it doesn't. Anyway idea is good and I like it so it can be proposed to our style guide.

Reply:
Conversation is closed due to inactivity and absence of rule that only it's author can close it.